packages/api/endpoints/reconciliation-reports.js

```diff
@@ -61,6 +74,7 @@ async function getReport(req, res) {
     }
     throw error;
   }
+  return res.boom.badImplementation('Should never get here');
```

Comment:
This should have a more user friendly message just in case.

Reply:
Well, it is a 500 so the user will only ever see {message: "Internal Error"}, but I can try to think of something else to put in there.